src/App.php

```diff
@@ -126,6 +126,51 @@ class App
 	 */
 	private $pConfig;
 
+	/**
+	 * Set the user ID
+	 *
+	 * @param int $user_id
+	 * @return void
+	 */
+	public function setUserId(int $user_id)
+	{
+		$this->user_id = $user_id;
+	}
+
+	/**
+	 * Set the nickname
+	 *
+	 * @param int $user_id
+	 * @return void
+	 */
+	public function setNickname(string $nickname)
+	{
+		$this->nickname = $nickname;
+	}
+
+	public function isLoggedIn()
+	{
+		return local_user() && $this->user_id && ($this->user_id == local_user());
+	}
+
+	/**
+	 * Fetch the user id
+	 * @return int 
+	 */
+	public function getUserId()
+	{
+		return $this->user_id;
+	}
+
+	/**
+	 * Fetch the user nick name
+	 * @return string
+	 */
+	public function getNickname()
```

Comment:
Is it the nick name of the App itself? If not, please make the name more specific for its intended use.

Reply:
You mean `getUserNickname`?